src/test/java/com/training/basetest/WebBaseTest.java

```diff
@@ -11,36 +15,40 @@
 import org.openqa.selenium.remote.RemoteWebDriver;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
 import java.net.MalformedURLException;
 import java.net.URL;
 
-public class WebBaseTest {
+public class WebBaseTest extends JsonReaderUtility {
```

Comment:
this is not required - this would make JsonReaderUtility the base parent, which would lead to incorrect design.
Rather create an object of this class and call that function